pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -1590,6 +1590,10 @@ func (c *Cloud) CreateDisk(volumeOptions *VolumeOptions) (KubernetesVolumeID, er
 	createAZ := volumeOptions.AvailabilityZone
```

Comment:
I would be better to rename `AvailabilityZone` to `AvailabilityZones` to make it clear what's expected here.